### DISCUSSION TITLE
Welcome to pyresparser Discussions!

### DISCUSSION THREAD

**Discussion by OmkarPathak** [OmkarPathak/pyresparser#65] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by akai07** [comment]
how to use a directory containing more than 1 resume  using cli https://github.com/OmkarPathak/pyresparser/blob/master/docs/cli.md#cli 
PLEASE HELP WRITING THE CODE TO PATH DIRECTORY

**Reply by OmkarPathak** [reply]
Hi @akai07 you can find the documentation for that here: https://omkarpathak.in/pyresparser/cli/#parsing_mutliple_resumes

**Comment by shubhi15** [comment]
How to run pyresparser on local for contributing more features in this library?

**Comment by MKchethan** [comment]
Hi Omkar, I have gone through the pyresparser documentation. I Implemented it and it is working good except it is not recognizing experience from the resumes. Can you please specify the format in order for resume parser to recognize the experience.

**Comment by ghost** [comment]
Hi Omkar, which version of python needs this to run the code.
I have python 3.11.8 but it creates the conflict while running the resumeparser.
while downloading spacy 2.1.4 version.
